lib/broadside/target.rb

```diff
@@ -74,14 +85,29 @@ def family
       "#{Broadside.config.application}_#{@name}"
     end
 
-    def to_h
+    def to_hash
```

Comment:
https://stackoverflow.com/questions/11182052/to-s-vs-to-str-and-to-i-to-a-to-h-vs-to-int-to-ary-to-hash-in-ruby